code/datums/jobs.dm

```diff
@@ -1487,6 +1487,31 @@ ABSTRACT_TYPE(/datum/job/civilian)
 		src.access = get_access("Mail Courier")
 		return
 
+/datum/job/special/stowaway
+	name = "Stowaway"
+	limit = 2
+	wages = 0
+	low_priority_job = TRUE
+	slot_back = list(/obj/item/storage/backpack/satchel/anello)
+	slot_belt = list(/obj/item/crowbar)
+	slot_foot = list(/obj/item/clothing/shoes/brown)
+	slot_jump = list(/obj/item/clothing/under/color/grey)
+	slot_head = list(/obj/item/clothing/head/green)
+	slot_ears = list(/obj/item/device/radio/headset/civilian)
+	slot_poc1 = list(/obj/item/currency/spacecash/fivehundred)
+	slot_poc2 = list(/obj/item/scissors)
+	slot_lhan = list(/obj/item/screwdriver)
+
+	items_in_backpack = list(/obj/item/currency/spacecash/buttcoin = 2)
+	rounds_needed_to_play = 10 // It would really suck ass to play this as a new player
+	// missing wiki link, does not have a mention on https://wiki.ss13.co/Jobs
+
+	special_setup(var/mob/living/carbon/human/M)
+		..()
+		if (!M)
+			return
+		M.traitHolder.addTrait("stowaway")
```

Comment:
Have you tested this works to spawn them in the right place? Roundstart init order is notoriously messy.

Reply:
I have!